scripts/docker-build.sh

```diff
@@ -18,11 +18,11 @@ docker rm -f $(docker ps -a -q -f name=${BUILD_CONTAINER_NAME} 2>/dev/null) 2>/d
 
 if docker image ls -f "reference=${BUILD_IMAGE_NAME}:${PG_IMAGE_TAG}" | grep "$BUILD_IMAGE_NAME" 2>/dev/null; then
     echo "Using existing build image ${BUILD_IMAGE_NAME}:${PG_IMAGE_TAG}"
-    docker run -d --name ${BUILD_CONTAINER_NAME} -v ${BASE_DIR}:/src ${BUILD_IMAGE_NAME}:${PG_IMAGE_TAG}
+    docker run --env TIMESCALEDB_TELEMETRY=off -d --name ${BUILD_CONTAINER_NAME} -v ${BASE_DIR}:/src ${BUILD_IMAGE_NAME}:${PG_IMAGE_TAG}
```

Comment:
Strictly, I don't think this needs the env variable since this is only a PostgreSQL build image. And, although TimescaleDB is later built in it, I don't think the extension is ever installed in the running container. I guess the env variable doesn't hurt though, although I am wondering if it is not better to add a line to the `postgresql.conf` file (like we do with `shared_preload_libraries`) while we are messing with the image anyway?

Reply:
Yes, that was how I turned off telemetry in `docker-build.sh` previously. But I removed it (and replaced with this env variable) for consistency, meaning, otherwise it wouldn't be clear to someone skimming the scripts why this particular `docker run` doesn't turn off telemetry; adding a line to `.conf` is buried deep in the `docker exec` line. I thought having both would be redundant, but I guess we could have both.

On a side note, I think I'll wait for your PR #778 to go in before rebase/merging this PR, since we are changing all the same files.

Reply:
I think the issue of having to pass this env variable to every place in every script that starts this image is very prone to errors. 

Setting this persistently in the image via the conf file only needs to be done at build time and we're done.